public/theme-overrides/styles.css

```diff
@@ -435,6 +435,19 @@ dl dd a {
   color: #303436;
 }
 
+/* Checkbox ***************************************/
+#hovered-checkbox + label:before {
```

Comment:
Why did you have to do this?

Reply:
Documentation purpose: it keeps the states of the checkboxes and prevents the user from changing the checkbox's state.